pkg/execution/executor/service.go

```diff
@@ -478,12 +479,13 @@ func (s *svc) scheduleGeneratorResponse(ctx context.Context, item queue.Item, r
 				return s.queue.Enqueue(ctx, item, time.Now())
 			case enums.OpcodeStep:
 				// Re-enqueue the exact same edge to run now.
-				if err := s.state.Scheduled(ctx, item.Identifier, edge.Edge.Incoming, 0, nil); err != nil {
+				if err := s.state.Scheduled(ctx, item.Identifier, "step", 0, nil); err != nil {
```

Comment:
Hmm, why is this hard-coded?  This is a generator edge, which means "replay the current edge" - edge.Edge.Incoming should be of type "step".

Reply:
👍 This can likely be removed - most probably a remnant of me throwing changes at it to see what breaks. 😄